doc/releases/releases.yml

```diff
@@ -15,6 +15,8 @@ releases:
   quincy:
     target_eol: 2024-06-01
     releases:
+      - version: 17.2.4  
+        released: 2022-09-30
```

Comment:
The build was failing and I couldn't figure out why, but it's because I put in Monday's date, which doesn't exist yet. So changing the date to today helped the build complete. This comment is here to remind us to change the date again on Monday.